tests/end-to-end/external-solution-integration/README.md

```diff
@@ -103,3 +103,47 @@ Currently, the test cannot be executed locally, as it requires access to interna
   ko github.com/google/ko
 
 >**TIP:** If you are running the test on a cluster with invalid or self-signed SSL certificates, use the `--skipSSLVerify` flag.
+
+### Debug it locally
+
+>**Requirements:** 
+  ko github.com/google/ko
+  dlv github.com/go-delve/delve
```

Comment:
Those are rendered in one line on github, Is this intended?

https://github.com/kyma-project/kyma/blob/c115aa1c10ca8f675e312df2a2a3547a239f3aa0/tests/end-to-end/external-solution-integration/README.md#run-the-test-locally

Reply:
No, thanks for letting me know.

Reply:
Fixed, thanks @alexandra-simeonova